kitsune/users/models.py

```diff
@@ -101,6 +101,7 @@ class Profile(ModelBase):
     fxa_avatar = models.URLField(max_length=512, blank=True, default="")
     products = models.ManyToManyField(Product, related_name="subscribed_users")
     fxa_password_change = models.DateTimeField(blank=True, null=True)
+    fxa_refresh_token = models.CharField(blank=True, default="", max_length=128)
```

Comment:
why do we need to save this in the model? The OIDC lib has a flag that will store the token if needed in the user's session. Can't we use that?

Reply:
the OIDC lib only stores the `access_token`, which expires after a few hours - we need the long-lived `refresh_token`
we also can't access the user's session when processing an incoming FxA event token